components/operator-service/src/main/java/com/wso2telco/dep/operatorservice/model/BlacklistWhitelistDTO.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) 2016, WSO2.Telco Inc. (http://www.wso2telco.com) All Rights Reserved.
+ * <p>
+ * WSO2.Telco Inc. licences this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.wso2telco.dep.operatorservice.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BlacklistWhitelistDTO {
+    private String apiID;
+    private String appId;
+    private String serviceProvider;
+    private List<String> msisdnList;
+    private String action;
+    private String user;
+
+    public BlacklistWhitelistDTO() {
```

Comment:
You do not need to write the default constructor here again. Since all the string values are set to null when declaring changing the 26th array definition line to "private List<String> msisdnList = new ArrayList<>();" is enough.

Reply:
done